### PR TITLE
Upgrade to gradle-commons 1

### PR DESCRIPTION
**Pull request by Azurelol**
## Description

Upgrades to latest gradle-commons.

## Changes
* ![UPDATE] Upgrade to `gradle-commons` `1`
* ![UPDATE] Upgrade to `plugins`  `2.2.4` since it was not building anymore (grgit issues)

[NEW]:https://atlas-resources.wooga.com/icons/icon_new.svg "New"
[ADD]:http://resources.atlas.wooga.com/icons/icon_add.svg "Add"
[IMPROVE]:http://resources.atlas.wooga.com/icons/icon_improve.svg "IMPROVE"
[CHANGE]:http://resources.atlas.wooga.com/icons/icon_change.svg "Change"
[FIX]:http://resources.atlas.wooga.com/icons/icon_fix.svg "Fix"
[UPDATE]:http://resources.atlas.wooga.com/icons/icon_update.svg "Update"

[BREAK]:http://resources.atlas.wooga.com/icons/icon_break.svg "Remove"
[REMOVE]:http://resources.atlas.wooga.com/icons/icon_remove.svg "Remove"
[IOS]:http://resources.atlas.wooga.com/icons/icon_iOS.svg "iOS"
[ANDROID]:http://resources.atlas.wooga.com/icons/icon_android.svg "Android"
[WEBGL]:http://resources.atlas.wooga.com/icons/icon_webGL.svg "Web:GL"

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -15,7 +15,7 @@
  */
 
 plugins {
-    id 'net.wooga.plugins' version '2.2.2'
+    id 'net.wooga.plugins' version '3.0.0-rc.2'
```
Comment (Larusso) [inline build.gradle:18]: Don't update this. This will create a group change! We wanted to do this as a seperate upgrade round

**Review by Larusso — CHANGES_REQUESTED** [review]

File: build.gradle (hunk 1 above)
Comment (Azurelol) [reply build.gradle:18]: Whoops.

**Review by Larusso — APPROVED** [review]

File: build.gradle (hunk 1 above)
Comment (Azurelol) [reply build.gradle]: Wait, I remember now why I did so in the first place. It wouldn't build until I updated it:

```
An exception occurred applying plugin request [id: 'net.wooga.plugins', version: '2.2.2']
> Failed to apply plugin class 'wooga.gradle.version.VersionPlugin'.
   > Could not create plugin of type 'VersionPlugin'.
      > Could not generate a decorated class for type VersionPlugin.
         > org/eclipse/jgit/errors/RepositoryNotFoundException has been compiled by a more recent version of the Java Runtime (class file version 55.0), this version of the Java Runtime only recognizes class file versions up to 52.0

Comment (Larusso) [reply build.gradle]: This is the grgit issue. I think the latest 2.x solves this. So you can upgrade to this.

Comment (Larusso) [reply build.gradle]: `2.2.4` that is

Comment (Azurelol) [reply build.gradle]: Yes, that fixed it. Thank you.

**Review by Larusso — APPROVED** [review]

**Pull request merged**